README.md

```diff
@@ -1,13 +1,11 @@
-# Porfolio: Application Deploy 
-A Microverse exercise to complete the Desktop version.
+# Porfolio: Mobile-menu
+A Microverse exercise to implement the hamberger of the mobile version
 <a name="readme-top"></a>
 
 <div align="center">
   <h3><b>Microverse README Template</b></h3>
```

Comment:
- [ ] As inidicated by the previous reviewer, kindly remove placeholder text. This include the Default title Microverse ReadMe Template. Remeber the objective is to personalize the ReadMe. 
![Screen Shot 2023-02-06 at 3 12 40 PM](https://user-images.githubusercontent.com/62849872/217108414-e94d1273-3187-4604-9197-675f08edbe96.png)